### DISCUSSION TITLE
Proposed DEI Initiative: Vision and Mission Statement

### DISCUSSION THREAD

**Discussion by amelia-gm** [o3de/community#85] [Diversity]
Hi all,

I'm starting this thread for us to chat about our mission and vision statements.

I'm attaching some of my notes from the meeting below.

The vision statement that we were talking about was something along the lines of "O3DE's community and projects will reflect the diversity of people on earth". 

Here's my notes:
1.	What outcome do we want?
2.	We want people to feel comfortable contributing
1.	Automatic Non-inclusive language flagging
3.	Folks don’t feel like they belong.
4.	Is the makeup of the community and leadership reflective of the people on the world?
5.	How we attract folks to be part of the project.
6.	Vision: Our Community and our project reflects the diversity of the world.
7.	We want it to be accessible by all.
8.	We want it to reflect greater diversity.
9.	We want the project to be inclusive in how it’s used
10.	Make sure events are safe
11.	Have opinionated statements of issues happening in the games industry

**Comment by hisergiorojas** [comment]
Proactively, recruiting new contributors that represent different demographics, viewpoints, opinions, and ideas.
